dll/ime/msctfime/lang/tr-TR.rc

```diff
@@ -0,0 +1,23 @@
+/*
+ * PROJECT:     ReactOS msctfime.ime
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Turkish resource file
+ * TRANSLATOR:  Copyright 2024 Ethem De Santa (scropool) <oyunyumru4@gmail.com>
+ */
+
+LANGUAGE LANG_TURKISH, SUBLANG_DEFAULT
+
+STRINGTABLE
+BEGIN
+    IDS_OK               "&Tamam"
+    IDS_CANCEL           "&İptal"
```

Comment:
None of existing translations use accelerator keys for these items.

Reply:
Good point. Is that a glitch in en-US? Or are the accels unneeded in tr-TR? What does MS do?

Reply:
I remember checking Windows in en-US and ru-RU locales, they were not underlined there.

Reply:
Thx. Then the turkish guys should strip all of them too!

Reply:
@binarymaster look at new commits.

Reply:
Resolved.